plugins/00_dokku-standard/commands

```diff
@@ -46,6 +47,51 @@ case "$1" in
         plugn trigger post-build-dockerfile "$APP"
         ;;
 
+      compose)
+        # extract first port from docker-compose.yml
+        COMPOSE_PORT=$(shyaml get-value app.ports.0 < docker-compose.yml 2> /dev/null || true)
+        [[ ! -n "$COMPOSE_PORT" ]] && COMPOSE_PORT=$(shyaml get-value web.ports.0 < docker-compose.yml 2> /dev/null || true)
```

Comment:
if shyaml comes from the python-yaml package then you'll want to add installation of this package to the Makefile.